spec/index.html

```diff
@@ -191,6 +191,28 @@ <h2>Introduction</h2>
     <strong>RDF Canonicalization algorithm version 1.0</strong> or
     <a>RDFC-1.0</a>.</p>
 
+  <div class="note">
+    <p>[[[RDF11-CONCEPTS]]] [[RDF11-CONCEPTS]] lacks clarity on the representation of
+      <a data-cite="RDF11-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a>,
+      where <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tags</a> of the form `xx-YY`
+      are treated as being case insensitive. Impelementations may represent language tags
+      using all lower case in the form `xx-yy`,
+      retain the original representation `xx-YY`,
+      or use [[BCP47]] <a data-cite="BCP47#section-2.1.1">formatting conventions</a>,
+      leading to different canonical forms, and therefore, different hashed values.</p>
+    <ul>
+      <li>The Canonicalization algorithm is based on the RDF 1.1 definition,
+        in the sense that the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> `xx-YY`
+        is case insensitive, which may lead to different canonicalizations if the user is not aware of this problem.</li>
+      <li>User communities should agree to use an agreed-upon choice between upper case or lower case
+        <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>,
+        while being aware that some implementations may normalize language tags,
+        affecting hash values.</li>
+      <li>Future evolution of RDF might regulate this issue, which RDF environments may have to adapt to,
+        and this may lead to an update of <a>RDFC-1.0</a>.</li>
```

Comment:
```suggestion
      <li>Future evolution of RDF might regulate this issue, which RDF environments might have to adapt to,
        and this might lead to an update of <a>RDFC-1.0</a>.</li>
```